textile/features.textile

```diff
@@ -1655,6 +1656,18 @@ h4. DevicePushDetails
 * @(PCP3)@ recipient - a map of string key/value pairs containing details of the push transport and address
 * @(PCP4)@ state - the state of the push registration, one of  @Active@, @Failing@, @Failed@
 
+h3(#introspection). Client library introspection
+
+h4. ClientRuntime
+* @(CR1)@ Provides information about the client library and the environment in which it’s running.
+* @(CR2)@ @agents@ static property:
+** @(CR2a)@ Returns a @Dict<String, String?>@ that lists the default key-value entries that the library uses to populate the @Agent@ library identifier, as described by "@RSC7d1@":#RSC7d1. An example would be @{ "ably-java": "1.2.1", "android": "24" }@.
+* @(CR3)@ @agentIdentifier@ static method:
+** @(CR3a)@ The client library must offer this method if and only if it offers the @ClientOptions#agents@ property described by "@RSC7d6@":#RSC7d6.
```

Comment:
I'm worried that we're inferring that the client library must implement this method if it's implemented `RSC7d6`. Perhaps clearer like this?:

```suggestion
** @(CR3a)@ The client library must only offer this method if it also offers the @ClientOptions#agents@ property described by "@RSC7d6@":#RSC7d6.
```

Reply:
Ah yes, that was bad wording on my part. I've changed it to your suggestion 👍